### PR TITLE
Homework 1 - Tianhong Yin

### PR DESCRIPTION
**Pull request by tyin29**
The first assignment was at first a bit hard for me to get started. I previously thought this class would be easy for me as I have already been familiar with R and R Markdown as well as techniques in basic statistical analysis. But I later realized that I was a bit naïve when I found it obscure to understand the instruction about Git and GitHub, which are what I did not learn before. I had no idea of what they are used for, let alone the terms (commit, pull, push, etc.) mentioned in the homework assignment. So before getting started, I read the instructions on the course website very carefully and repeated reading them to have a better understanding of the steps in the workflow, and I found it especially useful to watch the instruction video, from which I got a clear idea of what I was going to do.

I started the assignment by forking the hw01 repository and cloning the repository to my laptop after opening R Studio, which were easy to finish after watching the video tutorial. Then I clicked on README.md in the bottom right Files panel and started to modify the file locally by following the guide for writing Markdown documents (https://guides.github.com/features/mastering-markdown/) mentioned in the course instruction. Each time I modified the file, I would save the file, click on the Git tab in the upper right of R Studio, stage and commit the file. After finishing all edits, I clicked on the green button in the Git tab to push the committed changes to GitHub, which were later made visible online after I refreshed my page.

After understanding Git, GitHub, the terms of the technical actions, and how these things are connected, the assignment became not as hard as I thought at first. I just needed to be extremely careful to make sure each edit is right because if I make a mistake, the program would not do something that I wanted it to do. The only problem I got is that I was not sure about how to add a picture in the right way. But it was quickly solved after I resorted to Stack Overflow (https://stackoverflow.com/). There was a space between “]” and “(” in the syntax of inserting an image. Then I realized I should delete the space (the same for inserting a link). But I am still confused about which files to commit. For this assignment, I only need to commit the README.md file. But on the course website, there is an introduction of which files to commit, which makes me a bit confused.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,17 @@
...
+* Conflict Studies
+* American Foreign Policy
+* American Politics
+* Quantitative Methodology
+
+I am a cat person and I love to know everything about cats during my spare time. Here is the [link to a cat forum](http://thecatsite.com) I often visit to learn about cats.
+
+I also have two cats in my home back in China, and below is their photo.
+
+![Cat Photo](https://scontent-xsp1-2.xx.fbcdn.net/v/t1.0-9/92608591_228099475076501_1090134855175372800_n.jpg?_nc_cat=104&_nc_sid=110474&_nc_ohc=Mk8JVjPa3xoAX8DBPDu&_nc_ht=scontent-xsp1-2.xx&oh=d252350ea2fe8cfcfd2cefaf4f1956f0&oe=5EB8BA0A)
```
Comment (reed-diana) [inline README.md:17]: Nice use of images in the assignment!

File: README.md [hunk 2]
```diff
@@ -1 +1,17 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## **About Me**
+
+Hello, My name is **Tianhong Yin**, an MA student in international relations at the University of Chicago.
+
+My **research interests** include:
+* Conflict Studies
```
Comment (reed-diana) [inline README.md:8]: the unordered list works well here

File: README.md [hunk 3]
```diff
@@ -1 +1,17 @@
...
+* American Foreign Policy
+* American Politics
+* Quantitative Methodology
+
+I am a cat person and I love to know everything about cats during my spare time. Here is the [link to a cat forum](http://thecatsite.com) I often visit to learn about cats.
```
Comment (reed-diana) [inline README.md:13]: I like the way you added a link and renamed it for ease of use

File: README.md [hunk 4]
```diff
@@ -1 +1,17 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## **About Me**
```
Comment (reed-diana) [inline README.md:3]: The use of the header here is a good idea. This isn't really a critique, but you could include another header later in the file as well, maybe when you transition into talking about nonacademic interests.

File: README.md [hunk 5]
```diff
@@ -1 +1,17 @@
...
+
+My **research interests** include:
+* Conflict Studies
+* American Foreign Policy
+* American Politics
+* Quantitative Methodology
+
+I am a cat person and I love to know everything about cats during my spare time. Here is the [link to a cat forum](http://thecatsite.com) I often visit to learn about cats.
+
+I also have two cats in my home back in China, and below is their photo.
```
Comment (reed-diana) [inline README.md:15]: Another thing you could think about is resizing/cropping the image. I am not sure how to do that, though, other than cropping/resizing the source (I tried to do this in my own assignment.)

File: README.md (hunk 3 above)
Comment (reed-diana) [reply README.md:13]: One thing that's more typical style is to omit the "link to" part of the link text. You could just say "Here is a [cat forum] I go to . . .  
That's just a minor style option, though. I couldn't think of anything else. Looks great!

File: README.md (hunk 1 above)
Comment (djanvier) [inline README.md:17]: Cute!

File: README.md (hunk 3 above)
Comment (djanvier) [reply README.md:13]: I would echo this point, but ultimately as your peer said, it is a minor styling choice.

**Review by djanvier — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      x     |              |            |
| **Coding strategy**         |      x     |              |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |     x      |              |            |
| **Reproducibility**         |    x       |              |            |
* Good workflow and very thorough commit history!

File: README.md (hunk 3 above)
Comment (tyin29) [reply README.md:13]: Your suggestions are useful, and I will pay attention to that in future work. Thanks!

File: README.md (hunk 1 above)
Comment (tyin29) [reply README.md:17]: Thank you!

**Pull request closed without merging**